extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-aks.ipynb

```diff
@@ -331,7 +331,7 @@
                 "os.environ[\"AZDATA_USERNAME\"] = mssql_username\n",
```

Comment:
I see that the line above this is setting ACCEPT_EULA = 'yes'. Didn't we discuss this and determine that this should never be set by the notebook since the deploy wizard sets it itself? We should remove that if that's still the case.

Reply:
This is pre-existing code not part of this change. We can tackle fixing all those pre-existing ones as separate bug fix. For this work, I would like to keep it scoped.

Reply:
Tracking this issue with: https://github.com/microsoft/azuredatastudio/issues/8262